Dockerfile

```diff
@@ -4,7 +4,8 @@ FROM ubuntu:20.04 as builder
 RUN DEBIAN_FRONTEND="noninteractive" apt-get update
 RUN DEBIAN_FRONTEND="noninteractive" apt install -y cmake \
 clang-10 llvm-10 nasm ninja-build libnuma-dev \
-libcap-dev libglfw3-dev libepoxy-dev
+libcap-dev libglfw3-dev libepoxy-dev python3-dev \
+python3 linux-headers-5.8.0-23-generic
```

Comment:
Good catch.
Instead of explicitly defining the version number in the package, you should use Ubuntu's meta package of `linux-headers-generic`.
This will ensure that future updates from Canonical's repos don't break these rules.

Reply:
Thank you :)
Just updated the PR to use the meta package for headers.